apps/web/components/OfficialJournalOfIceland/OJOIAppendix.tsx

```
@@ -0,0 +1,45 @@
+          return (
+            appendix.html && (
+              <div id={id} key={id}>
+                <AccordionItem
+                  id={id + '-internals'}
+                  labelVariant="h4"
+                  labelUse="h3"
+                  label={appendix.title}
+                >
+                  <Box
+                    className={s.bodyText}
+                    dangerouslySetInnerHTML={{ __html: appendix.html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)